core/base/inc/Rtypes.h

```diff
@@ -263,13 +263,14 @@ class ClassDefGenerateInitInstanceLocalInjector:
 }} // namespace ROOT::Internal
 
 
-// Common part of ClassDef definition.
+// Common part of ClassDef definition, used both by ClassDef and ClassDefInline.
```

Comment:
Technically it is `ClassDefOutline and ClassDefInline` (direct use) or a long list of ClassDef variation (indirectly).

Reply:
Fixed.